server/src/main/java/org/opensearch/search/profile/Timer.java

```diff
@@ -71,6 +71,10 @@ public final void start() {
         doTiming = (count - lastCount) >= Math.min(lastCount >>> 8, 1024);
         if (doTiming) {
             start = nanoTime();
+            if (isStarted == false) {
```

Comment:
Please remove `isStarted` flag, it does not play well with a) doTiming b) count c) stop() implementation